Contributors.md

```diff
@@ -0,0 +1,68 @@
+<p align="center">
+<a href="https://dscvit.com">
+	<img src="https://user-images.githubusercontent.com/30529572/72455010-fb38d400-37e7-11ea-9c1e-8cdeb5f5906e.png" />
+</a>
+</p>
+
+## Contributors
+
```

Comment:
Why can't you add these in the main readme only in the same template, including it in a separate readme seems to be useless because there is nothing else to show!

Reply:
The Contributors.md was an idea suggested by Angad during the Last Digital Martketing meeting. 

Should I merge it into the main file anyways?

Reply:
The Contributors.md was an idea suggested by Angad during the Last Digital Marketing meeting.

Should I merge it into the main file anyways?

Reply:
Yeah merge it!